app/services/calendar_service.rb

```diff
@@ -0,0 +1,40 @@
+class CalendarService
+
+  def self.post_events(user, title, description, date)
+    response = conn.post('event/new?') do |req|
+      req.params['token'] = user.google_token
+      req.params['refresh_token'] = user.google_refresh_token
+      req.params['calendar_id'] = user.calendar_id
```

Comment:
the `calendar_id` is now passed in the request instead of the `name` of the `calendar` so that we can target the `calendar` of the `user`'s account even if the name of the calendar is changed by the `user` after having signed in the first time.

Reply:
Goo call on this